README.md

```diff
@@ -155,31 +183,47 @@ the order of tags must exactly be the same with that defined in the schema.
 And the non-existing tags must be fulfilled (with NullValue) instead of compacting all non-null tag values.
 
 ```java
-StreamWrite streamWrite = StreamWrite.builder()
-    .elementId(segmentId)
-    // write binary data to "data" tag family
-    .dataTag(Tag.binaryField(byteData))
-    .timestamp(now.toEpochMilli())
-    .name("sw")
-    // write indexed tags to "searchable" tag family
-    .tag(Tag.stringField(traceId))
-    .tag(Tag.stringField(serviceId))
-    .tag(Tag.stringField(serviceInstanceId))
-    .tag(Tag.stringField(endpointId))
-    .tag(Tag.longField(latency))
-    .tag(Tag.longField(state))
-    .tag(Tag.stringField(httpStatusCode))
-    .tag(Tag.nullField())
-    .tag(Tag.stringField(dbType))
-    .tag(Tag.stringField(dbInstance))
-    .tag(Tag.stringField(broker))
-    .tag(Tag.stringField(topic))
-    .tag(Tag.stringField(queue))
-    .build();
+StreamWrite streamWrite = new StreamWrite("default", "sw", segmentId, now.toEpochMilli(), 1, 13)
+    .dataTag(0, Value.binaryTagValue(byteData))
```

Comment:
`StreamWriter` should not specify only two tag families, `data` and `searchable` since users have the capacity to pass the tag family name to the client in schema management.